doc/tutorials/basic/build_dashboard.md

```diff
@@ -0,0 +1,306 @@
+# Build a Dashboard
+
+In this tutorial we will build a beautiful dashboard showing key metrics of wind turbine manufacturers.
```

Comment:
I believe this comment is here because you read the markdown file instead of the rendered tutorial. The app is actually embedded. Thus is should be very clear what the user is creating. 

![image](https://github.com/holoviz/panel/assets/42288570/1856c793-905e-4a7c-a75a-ca6b0c7dff4f)

Feel free to reopen and describe more specifically what needs to be expanded upon.

Reply:
I've updated to 

Welcome to our tutorial on building an interactive dashboard showcasing essential metrics from wind turbine manufacturers.

We'll demonstrate how to leverage a variety of components such as sliders, dropdowns, plots, indicators, tables and layouts to craft a visually stunning and functional application.

By following along, you'll gain insights into how these components can be seamlessly combined to present data in a meaningful way.

Our objective is to empower you to replicate this process for your own datasets and use cases, enabling you to create impactful dashboards tailored to your specific needs.